eco.Rmd

```diff
@@ -25,42 +25,47 @@ finally, point to books on ecological modeling especially emphasizing Alain Zuur
-We will use ordination techniques to extract the floristic gradient developing along the southern mountain slope.
-We expect the ordination to distinguish four plant belts along the altitudinal gradient: a low-elevation *Tillandsia* belt, a herbaceous belt, a bromeliad belt, and an uppermost succulent belt [@muenchow_soil_2013].
-Secondly, we would like to model the scores of the first ordination axis with environmental predictors such as slope, catchment area, altitude and NDVI.
-The resulting model shall serve to make spatial predictions of the floristic composition.
-When doing so, we will account for over-optimistic predictions in the presence of spatial autocorrelation using spatial cross-validation.
+During a field study to Mt. Mongón we recorded the plants living in 100 randomly sampled 4x4 m^2^ plots in the austral winter of 2011 [@muenchow_predictive_2013], which coincided with a strong La Niña weather pattern that year.
+This led to even higher levels of aridity than usual in the coastal desert, while also increasing fog activity on the *lomas* mountains of northern Peru.
+
+Ordination techniques are ... and will allow us to extract the floristic gradient developing along the southern mountain slope.
+The first hypothesis is that four plant belts will be found along the altitudinal gradient: a low-elevation *Tillandsia* belt, a herbaceous belt, a bromeliad belt, and an uppermost succulent belt [@muenchow_soil_2013].
+The second is that the first 'ordination axis' is related to environmental predictors such as slope, catchment area, altitude and NDVI.
+The result will be a catchment model, which can be used to make spatial predictions of about the composition of plants anywhere in the study area based on a relatively small sample.
+Further, we will account for spatial autocorrelation using spatial cross-validation.
 
 ## Reducing dimensionality
+
 Ordinations are a popular tool in vegetation science to extract the main information, frequently corresponding to ecological gradients, from large species-plot matrices mostly filled with 0s. 
 However, they are also used in remote sensing (spectral bands + hyperspectral), the soil sciences, geomarketing, etc..
+
 Principal component analysis (PCA) is probably the most famous ordination technique. 
 It is a great tool to reduce dimensionality if one can expect linear relationships between variables, and if the joint absence of a variable (for example calcium) in two plots (observations) can be considered a similarity.
 This is barely the case with vegetation data.
+
 For one, relationships are usually non-linear along environmental gradients.
 That means the presence of a plant usually follows a unimodal relationship along a gradient (e.g., humidity, temperature or salinity) with a peak at the most favorable conditions and declining ends towards the unfavarable conditions. 
+
 Secondly, the joint absence of a species in two plots is often hardly an indication for similarity.
```

Comment:
This all looks good but I think you should try to get onto the geocomputation quick and reduce the word count of this (admittedly very interesting) introduction if possible.